### DISCUSSION TITLE
How to achieve Handwriting/drawing React Native App?

### DISCUSSION THREAD

**Discussion by MuhammadRafeh** [software-mansion/react-native-reanimated#2548] [Ideas]
I am working on the App, in which I want to make feature in which user can draw with his fingers on the screen? How to achieve this. I just want to get idea for it?

@wcandillon Kindly help me Sir?

**Comment by numandev1** [comment]
@MuhammadRafeh you can use this package for drawing https://github.com/terrylinla/react-native-sketch-canvas
